src/constraints.jl

```diff
@@ -172,6 +172,22 @@ function delete(model::Model, con_ref::ConstraintRef{Model})
     MOI.delete(backend(model), index(con_ref))
 end
 
+"""
+    delete(model::Model, con_refs::Vector{ConstraintRef})
+
+Delete the constraints associated with `con_refs` from the model `model`.
+Solvers may implement methods for deleting multiple constraints that are
+more efficient than repeatedly calling the single constraint delete method.
```

Comment:
It's important to note that solvers are only expected to implement more efficient fallbacks when the element type of the index vector is concrete ([docs](http://www.juliaopt.org/MathOptInterface.jl/dev/apireference/#MathOptInterface.delete-Tuple{MathOptInterface.ModelLike,Array{#s1,1}%20where%20#s1%3C:Union{VariableIndex,%20ConstraintIndex}})). I.e., all constraints referenced are of the same function-in-set type.

Reply:
Is this a good wording?

> Solvers may implement specialized methods for deleting multiple constraints of
> the same concrete type. These may be more efficient than repeatedly calling the
> single constraint delete method.

Reply:
Yes, sounds good.

Reply:
Will be done then.